cmds/dhclient/dhcp6client/dhcp6client_test.go

```diff
@@ -0,0 +1,329 @@
+// Command dhcp6d is an example DHCPv6 server.  It can only assign a
```

Comment:
I forgot to put in the information about where this server code came from, we will need to do add it.